it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala

```diff
@@ -275,7 +274,10 @@ class DraftExciseMovementControllerItSpec extends PlaySpec
 
   private def assertValidResult(result: WSResponse) = {
     val responseBody = Json.parse(result.body).as[ExciseMovementResponse]
-    responseBody mustBe ExciseMovementResponse("Accepted", "LRNQA20230909022221", consignorId, Some("GBWKQOZ8OVLYR"))
+    responseBody.status mustBe "Accepted"
```

Comment:
Should we also be checking for the MovementId here?

Reply:
It is not super easy to, as we're generating the UUID in the apply method. But besides, because we are generating it, we feel like testing it here is either testing the type system, or testing the UUID library. Neither of which feel particularly valuable.

Reply:
Can we just check that this comes back and matches a regex for a uuid? This test would have flagged up a failure for the above comment (i.e. the response having the nested structure rather than just a string)